affiliate/src/productCarousel/components/IntegrationModal.jsx

```diff
@@ -0,0 +1,51 @@
+import './integrationModal.scss';
+
+import { Modal } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+const IntegrationModal = ({
```

Comment:
Code re-use, but afaik, it's tricky to share code among these Gutenberg blocks, so it might be better for now to make changes to one "master" components directory and then copy/paste them into the other blocks until a better way is found 🤷

Reply:
Does it not work pulling this component into a "shared" directory? It seems to work for me, but I could definitely be missing something. See comment I'm about to add below.